app/models/concerns/legal_aid_application_state_machine.rb

```diff
@@ -88,7 +88,8 @@ module LegalAidApplicationStateMachine
       end
 
       event :submit_assessment do
-        transitions from: :checked_merits_answers, to: :assessment_submitted
+        transitions from: :checked_merits_answers, to: :assessment_submitted,
```

Comment:
The only thing that worries me here is that the application is transitioned to `assessment_submitted`, before the pdfs are actually generated.  If there is a problem with the generation, or the background job system isn't available for example, we will have an application which is ostensibly complete, but no means/merits reports for them.

I t hink it would be better to have another state, `generating_merits_report` and transition to that when we fire off the background jobs, and have the `MeritsReportCreator`  transition to `assessment_submitted` once it's complete.